Cubical/Codata/Conat/Properties.agda

```diff
@@ -83,6 +83,24 @@ n+′∞≡∞′ (suc n) = λ i → suc (n+∞≡∞ n i)
 ∞+∞≡∞ : ∞ + ∞ ≡ ∞
 force (∞+∞≡∞ i) = suc (∞+∞≡∞ i)
 
++-zeroˡ : ∀ n → conat zero + n ≡ n
```

Comment:
I guess you can use `𝟘` for `conat zero`

Reply:
Done!